myproject/urls.py

```diff
@@ -19,6 +19,7 @@
 from boards import views
 
 urlpatterns = [
-	url(r'^homepage/$',views.home,name='home'),    
+	url(r'^homepage/$', views.home, name='home'),
+	url(r'^boards/(?P<pk>\d+)/$', views.board_topics, name='board_topics'),
```

Comment:
This actually comes from an old django version, <3.0 for sure. The syntax changes a bit to `path` instead of `url`